fl_training.py

```diff
@@ -142,6 +176,13 @@ def compute_test_score(
         global_val_acc = []
         global_val_loss = []
 
+        # Create list of weights for aggregation steps
+        aggregation_weights = []
```

Comment:
I think the `aggregation_weights`  should sum to 1:

- uniform:` [1/nodes_count] * nodes_count`
- data-volume: `[len(node.x_train) for node in node_list] / total_data_size_in_nodes`

et un petit `assert(np.sum(aggregation_weighting ) == 1) `à la fin

Reply:
Ok, doing that now.

Reply:
Done in cb625f7.